openhtf/io/test_record.py

```diff
@@ -29,6 +29,7 @@
 from openhtf.util import logs
 
 _LOG = logging.getLogger(__name__)
+CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
```

Comment:
No, those are parsed too late for this, I think. And it tends to be a
program decision rather than a per-invocation decision anyway

On Jul 16, 2016 3:15 PM, "grybmadsci" <notifications@github.com> wrote:

> In openhtf/io/test_record.py
> <https://github.com/google/openhtf/pull/363#discussion_r71072641>:
>
> > @@ -29,6 +29,7 @@
> >  from openhtf.util import logs
> >
> >  _LOG = logging.getLogger(__name__)
> > +CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
>
> hm, shouldn't this be a commandline arg?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/openhtf/pull/363/files/d8a9785b066f7349227d1a8561cf4173695026f3#r71072641>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASrtPvAY04Ja6Ms_qg9tq7gid0p1DeHks5qWVfwgaJpZM4JNtjc>
> .
>